homework/tables.sql

```diff
@@ -0,0 +1,63 @@
+CREATE DATABASE star;
+\c star
+
+CREATE TABLE products (
+product_code VARCHAR PRIMARY KEY,
+product_name VARCHAR,
+product_line VARCHAR,
+product_scale VARCHAR,
+product_vendor VARCHAR,
+quantity_in_stock INTEGER,
+buy_price MONEY,
+_m_s_r_p MONEY
+);
+--removed product_description
+--removed html_description
+
+CREATE TABLE customers(
+customer_number INTEGER PRIMARY KEY,
+customer_name VARCHAR,
+city VARCHAR,
+state VARCHAR,
+country VARCHAR, 
+credit_limit MONEY
+);
+--removed contact_last_name and contact_first_name
+--removed customer_location
+
+CREATE TABLE employees(
+sales_rep_employee_number INTEGER PRIMARY KEY,
+last_name VARCHAR,
+first_name VARCHAR,
+office_code INTEGER,
+city VARCHAR,
+country VARCHAR
+);
+--removed state: all in different states and some do not have states; analysis by city suffices
+--removed office_location: city/country is sufficient for analysis
+
+CREATE TABLE orders(
```

Comment:
Interesting. Not what I expected, but I can see why you did this. In general, the reason I don't love this is because "orders" here is really a mix of "orders" and "dates". It's quite standard in a Star Schema / Dimensional Modeling framework to have a "dates" dimension, so I think that's a good one to understand and make sure that if you break from that you have a good reason to do it. Date's provide a nice level of normalization, it normalizes a little (multiple things will have the same date it), whereas allows the dimensions of the dates (month, quarter, etc.) to be fully unnormalized.